Gravity-tube/src/Contexts/StateProvider.js

```diff
@@ -1,11 +1,14 @@
 import { AuthProvider } from "./Auth-context";
+import { FilterProvider } from "./Filter-context";
 import { PlaylistProvider } from "./Playlist-context";
 import { VideoProvider } from "./Videos-context";
```

Comment:
Suggestion - Instead of importing contexts separately, you can export them from a common file and import it here.

Reply:
ok sure